react/components/WithoutSSR.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { ComponentType } from 'react'
+import { NoSSR } from 'vtex.render-runtime'
+
+export default function(ChildComponent: ComponentType) {
+  return function WithoutSSR(props: any) {
+    return (
+      <NoSSR>
```

Comment:
When should I use this exactly?

Reply:
When you don't want something to be rendered on the server, i.e, a page which depends on a `query string` paramenter, such as the `order-placed` page 🤓

Reply:
Ended removing it in favour of adding `render: client` to the `interfaces.json`